website/src/pages/index.tsx

```diff
@@ -646,7 +641,7 @@ function Pods() {
 export default function Home(): JSX.Element {
   return (
     <Layout
-      title="Leverage Podman with a Desktop App"
+      title="Podman Desktop - A graphical tool for developing on containers and Kubernetes"
```

Comment:
AFAIK long title don't work (as we don't see the end)

![image](https://user-images.githubusercontent.com/436777/204256523-4009a746-8312-4c63-9784-9e5497a8a8db.png)

Reply:
Updated.